lib/rucio/api/rse.py

```diff
@@ -117,6 +118,17 @@ def del_rse_attribute(rse, key, issuer):
 
     return rse_module.del_rse_attribute(rse=rse, key=key)
 
+def reset_rse_checksum_support(rse, issuer):
```

Comment:
I see. I just wanted to avoid errors in the key spelling: isn't leaving it free a bit of an issue?
Same for the following :)

Reply:
I get the concern, but all these settings are to be set by admins anyway, so the knowledge should be there. You cannot stop them from using `rucio-admin rse set-attribute` anyway, even if the special method is there. And currently, for all the other functional attributes there is no special method, so most admins would not expect one for this as well. So to me this is more about documenting this properly. 
Anyway, I see this more as a "special" setting anyway, because for most RSEs you won't need it, only for very few "special" ones.

Reply:
Roger that then :)